### ISSUE TITLE
Improve startup feedback when container is being pulled

### ISSUE THREAD

**Issue by afscrome** [dotnet/aspire#3358]
When a container is being pulled for the first time, the resource remains in the "starting" state on the aspire dashboard, without any indication of why the resource is taking so long to start up.  That can sometiems take a long time for a large image and/or slow network connection.

It would be helpful to have some indication that the image is being pulled - perhaps something in the resource logs?

**Comment by davidfowl** [comment]
I agree. This known but I think it's out of scope for GA. Right now it just shows Starting...

**Comment by dbreshears** [comment]
DCP work is completed in 0.4.0 version

**Comment by davidfowl** [comment]
Do we get docker pull logs now?

**Comment by dbreshears** [comment]
Once the work  in the app-model / dashboard is completed

**Comment by davidfowl** [comment]
@dbreshears can you clarify what work needs to be done?

**Comment by danegsta** [comment]
DCP now supports an additional set of log types for a resource; `startup_stdout` and `startup_stderr`, in addition to the existing `stdout` and `stderr` feeds. Additionally, even if `follow` logs are requested for a startup feed, the stream will still close once all logs are sent (startup is complete), so it'll be possible to either show the startup logs as a separate stream or injected at the start of the normal log feed.

**Comment by davidfowl** [comment]
This is fixed in https://github.com/dotnet/aspire/pull/4601
